package.json

```diff
@@ -122,6 +121,8 @@
     "find-remove": "1.2.1",
     "fs-extra-promise": "^0.3.1",
     "getmac": "1.2.1",
+    "gulp": "^4.0.0",
```

Comment:
What are these changes for?

Reply:
gulp shouldn't be a runtime dependency so not sure why you moved the stuff down here.